src/staging/staging_main.py

```diff
@@ -270,11 +267,13 @@ def run_staging(  # noqa: C901
         if config["global"]["output_pnp_full_responses"]:
             StagingMainLogger.info("Starting output of staged PNP data...")
             staging_folder = staging_dict["staging_output_path"]
-            tdate = datetime.now().strftime("%y-%m-%d")
+            tdate = config["filename_items"]["tdate"]
+            run_id = config["filename_items"]["run_id"]
             survey_year = config["survey"]["survey_year"]
             survey_type = config["survey"]["survey_type"]
             staged_filename = (
-                f"{survey_type}_{survey_year}_staged_PNP_full_responses_{tdate}_v{run_id}.csv"
+                f"{survey_type}_{survey_year}_staged_full_responses_"
```

Comment:
Could we use the `filename_amender` function here instead? Would go from 5 lines long to 4?

Reply:
this issue is addressed in another branch that's to be merged.